src/phoenix/trace/tracer.py

```diff
@@ -17,26 +28,44 @@ def export(self, _: Any) -> None:
 
 
 _DEFUNCT_MSG = (
-    "DEFUNCT: `Tracer` is a defunct class in the current version of Phoenix. "
-    "It no longer has any purpose or functionality and will be removed in the future."
+    "DEFUNCT: `Tracer` is now a defunct class in the current version of Phoenix and is being "
+    "deprecated. It no longer has any purpose or functionality and will be removed in the future."
 )
+
 _USE_ENV_MSG = (
-    "Setting endpoint through the HttpExporter is no longer supported. "
-    'Use environment variables instead, e.g. os.environ["PHOENIX_PORT"] = "54321"'
+    "Setting the Phoenix endpoint through the HttpExporter is no longer supported. "
+    'Please use environment variables instead, e.g. os.environ["PHOENIX_PORT"] = "54321"'
+)
+
+_ON_APPEND_DEPRECATION_MSG = (
```

Comment:
this is redundant because the user already gets the message that the class is defunct when they try to initialize it

Reply:
I think it's helpful to clarify to the user that this argument doesn't do anything, even if the method is deprecated.